lib/httpClient.ts

```diff
@@ -4,13 +4,15 @@ import { Signer, Wallet } from 'ethers'
 import { JsonRpcProvider } from 'ethers/providers'
 import { arrayify, keccak256 } from 'ethers/utils'
 import { Claim, IRole } from './LoginStrategy.types'
+import { Policy } from 'cockatiel'
```

Comment:
I'm not too familiar with retry policy libraries in javascript ecosystem, is cockatiel one of the most popular? (I know Polly is very popular in .NET, which cockatiel seems to be based off of)

Reply:
Yeah it's pretty much popular, we're using at TSH this library for production ready apps :)